server/live-loan-router.js

```diff
@@ -185,6 +185,13 @@ module.exports = function liveLoanRouter(cache) {
 		});
 	});
 
+	// Loan Id IMG Router (Kiva Classic)
+	router.use('/lid/:id([a-zA-Z0-9.%,_-]{0,})/img2', async (req, res) => {
+		await tracer.trace('live-loan.loanid.serveImg', { resource: req.path }, async () => {
+			await serveImg('loanid', 'classic', cache, req, res);
+		});
+	});
```

Comment:
This is looking great! Please add similar `lid` route handlers for /img and /url